operator/src/main/java/oracle/kubernetes/operator/Main.java

```diff
@@ -526,21 +523,21 @@ private NamespaceWatcher createNamespaceWatcher(String initialResourceVersion) {
     return NamespaceWatcher.create(
         threadFactory,
         initialResourceVersion,
-        Namespaces.getLabelSelectors(),
         TuningParameters.getInstance().getWatchTuning(),
         this::dispatchNamespaceWatch,
         new AtomicBoolean(false));
   }
 
   void dispatchNamespaceWatch(Watch.Response<V1Namespace> item) {
```

Comment:
The `metadata` variable is not used after this check, meaning that you can simplify it to a single expression that returns the namespace. The computation is now complex enough to warrant being moved into its own method.